_dashboards/search-telemetry.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Search telemetry
+nav_order: 30
+---
+
+
+# About Search telemetry
+
+Search Telemetry is used to analyze performance for success or failed search requests in OpenSearch Dashboards. Telemetry data is saved in the `.kibana_1` index.
```

Comment:
Nit: maybe start these sentences with "Use search telemetry to analyze..." and "OpenSearch stores telemetry data in...", to make them active voice.

Reply:
ok thanks